src/app/application.cpp

```diff
@@ -495,7 +497,22 @@ int Application::exec(const QStringList &params)
     GuiIconProvider::initInstance();
 #endif
 
-    BitTorrent::Session::initInstance();
+    try {
```

Comment:
Maybe we should extend this `try` block to cover the initialization of all components? Some of them can also throw an exception when they are failed to create.

Reply:
Also handle the similar exceptions from RSS component now.